dashboard/index.html

```diff
@@ -6,149 +6,46 @@
       body {
         color: #333;
         font-size: 12px;
-        padding: 5px;
-      }
-
-      .bars-heading {
-        margin: 0 0 5px 0;
+        padding: 0;
       }
-      .bars-container {
-        border-bottom: 2px #d3d3d3 solid;
-        margin-top: 20px;
+      .charts {
         padding: 10px;
       }
-      .bars {
```

Comment:
does the google chart lib provide default styles then? if so, how is the css being loaded?